pkg/catalog/ingress.go

```diff
@@ -50,3 +53,57 @@ func (mc *MeshCatalog) GetIngressRoutesPerHost(service service.MeshService) (map
 
 	return domainRoutesMap, nil
 }
+
+// GetIngressPoliciesForService returns a list of inbound traffic policies for a service as defined in observed ingress k8s resources.
+func (mc *MeshCatalog) GetIngressPoliciesForService(svc service.MeshService, sa service.K8sServiceAccount) ([]*trafficpolicy.InboundTrafficPolicy, error) {
+	inboundIngressPolicies := []*trafficpolicy.InboundTrafficPolicy{}
+	ingresses, err := mc.ingressMonitor.GetIngressResources(svc)
+	if err != nil {
+		log.Error().Err(err).Msgf("Failed to get ingress resources for service %s", svc)
+		return inboundIngressPolicies, err
+	}
+	if len(ingresses) == 0 {
+		log.Trace().Msgf("No ingress resources found for service %s", svc)
+		return inboundIngressPolicies, err
+	}
+
+	defaultWildcardRoute := trafficpolicy.HTTPRouteMatch{
+		PathRegex: constants.RegexMatchAll,
+		Methods:   []string{constants.RegexMatchAll},
```

Comment:
How come this is `constants.RegexMatchAll` rather than `constants.WildcardHTTPMethod`

https://github.com/openservicemesh/osm/blob/main/pkg/catalog/routes.go#L25-L28
https://github.com/openservicemesh/osm/blob/af93fe0fb6395710341d5b91e20b04b4a06a9dbf/pkg/constants/constants.go#L87-L91

Reply:
Good catch, I have updated it to use `constants.WildcardHTTPMethod`

Reply:
cool I think this var becomes the same one used in the routes.go file at that point (wildCardRouteMatch) if you want to use that https://github.com/openservicemesh/osm/blob/main/pkg/catalog/routes.go#L25

Reply:
Thanks for noticing, I have updated the code